waterlog/src/Components/Loader.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const Loader = () => {
+    return(
+      <div>
+        <div className="lds-spinner">
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
```

Comment:
Why so many "div" ?

Reply:
this represents the bars on the loader
![loader](https://user-images.githubusercontent.com/46716201/52620271-c4a65b00-2eac-11e9-8359-b6983c118c29.PNG)

Reply:
Think you can do a loader with out so many divs: https://www.w3schools.com/howto/howto_css_loader.asp